### PR TITLE
stage6: Add eepromutils

### PR DESCRIPTION
**Pull request by mirceacaprioru**
These utilities are used to create, flash and dump HAT EEPROM images.

Signed-off-by: Mircea Caprioru <mircea.caprioru@analog.com>

### CODE REVIEW

File: stage6/08-eepromutil/00-run.sh [hunk 1]
```diff
@@ -0,0 +1,23 @@
+#!/bin/bash -e
+
+on_chroot << EOF
+
+mkdir -p "home/${FIRST_USER_NAME}"
+pushd "home/${FIRST_USER_NAME}"
+
+[ -d hats" ] || {
```
Comment (nunojsa) [inline stage6/08-eepromutil/00-run.sh:8]: is this on purpose? or did you meant `[ -d "hats" ]`?

**Review by nunojsa — COMMENTED** [review]
minor comment

File: stage6/08-eepromutil/00-run.sh (hunk 1 above)
Comment (mirceacaprioru) [reply stage6/08-eepromutil/00-run.sh:8]: missed that. will add it.

**Comment by mirceacaprioru** [comment]
changelog v2
- added missing "

File: stage6/08-eepromutil/00-run.sh (hunk 1 above)
Comment (nunojsa) [reply stage6/08-eepromutil/00-run.sh]: in the future if we know we are using bash, we can just use `[[ ]]` and safely forget about the quotes

**Review by nunojsa — APPROVED** [review]

**Comment by mirceacaprioru** [comment]
will initiate a build tonight to see if there aren't any issues. Will merge tomorrow if everything is ok.

**Pull request merged**